packages/browser-wallet/src/popup/pages/Account/SendCcd/CreateTransfer.tsx

```diff
@@ -26,11 +26,10 @@ import {
 import { useLocation, useNavigate } from 'react-router-dom';
 import { useAccountInfo } from '@popup/shared/AccountInfoListenerContext';
 import { useSelectedCredential } from '@popup/shared/utils/account-helpers';
-import { CCD_METADATA, getTokenBalance, getTokenTransferEnergy, TokenIdentifier } from '@shared/utils/token-helpers';
+import { CCD_METADATA, getTokenTransferEnergy, TokenIdentifier } from '@shared/utils/token-helpers';
```

Comment:
Should we use `CCD_METADATA` in a more safe manner? maybe hardcode the contract index for testnet, and only fall back if the index matches that?

Reply:
CCD_METADATA is trick so that I always have a metadata object, even when my "token" is the native CCD.

Reply:
Ah, i mistook it for the mock wccd metadata.. Sorry